rpc/send.go

```diff
@@ -151,9 +151,7 @@ func Send(opts Options, method string, addrs []net.Addr, getArgs func(addr net.A
 				continue
 			}
 			reply := getReply()
-			if log.V(1) {
```

Comment:
This isn't just a cosmetic change, btw. These two formulations do something differently. In the original, only the log level is consulted. In the second, clients[index].Addr() is invoked. Not sure if it matters here, but something to keep in mind. Using:

    if log.V(1) {
      ...
    }

Can be a lot more efficient at times.